src/sass/_theme_badge_fa.sass

```diff
@@ -1,8 +1,11 @@
 // Slimmer version of FA for use on the badge_only.sass file.
 
-+font-face(FontAwesome, '#{$fa-font-path}/fontawesome-webfont')
++font-face(FontAwesome, '#{$fa-font-path}/fa-solid-900')
 
-.fa:before
+.fas
```

Comment:
We probably want to keep this `fa` or this is a breaking change as well. Downstream projects could be relying on `fa` classes and the rename seems easy enough to avoid